liquibase-core/src/main/java/liquibase/command/AbstractCliWrapperCommandStep.java

```diff
@@ -2,12 +2,16 @@
 
 import liquibase.Scope;
 import liquibase.exception.CommandExecutionException;
+import liquibase.exception.CommandValidationException;
 import liquibase.integration.commandline.Main;
+import liquibase.license.LicenseServiceUtils;
 
 import java.io.OutputStream;
 import java.io.PrintStream;
 import java.util.*;
 
+import static liquibase.command.core.UpdateCommandStep.ROLLBACK_ON_ERROR;
```

Comment:
This class can't have a dependency on UpdateCommandStep because this is more of an "api" class vs. UpdateCommandStep that's an implementation thing.

Reply:
Cool, I didn't know that, I'm gonna create a private constant then. Thanks!